docker-compose.yml

```diff
@@ -1,30 +1,41 @@
 version: '3.8'
 services:
-  tamashi-bot:
-    build: ./
-    image: rofedl/tamashi:1.2.7
-    ports:
-      - 8080:8080
-    depends_on:
-      - redis
-      - lavalink
-    networks:
-      - default
-    environment:
-      REDIS_DOCKER_HOST: redis
-      LAVALINK_DOCKER_HOST: lavalink
-    volumes:
-      - type: bind
-        source: ./config.env
-        target: /opt/tamashi/config.env
+  # tamashi:
+  #   build: ./
+  #   container_name: tamashi_bot
+  #   image: rofedl/tamashi:2.0.0
+  #   ports:
+  #     - 8080:8080
+  #   depends_on:
+  #     - redis
+  #     - lavalink
+  #   networks:
+  #     - default
+  #   environment:
+  #     REDIS_DOCKER_HOST: redis
+  #     LAVALINK_DOCKER_HOST: lavalink
+  #   volumes:
+  #     - type: bind
+  #       source: ./src/config.json
+  #       target: /opt/tamashi/src/config.json
```

Comment:
What happened to this?

Reply:
Haven't decided if I wanna containerize the bot as well, might do it later down the line but if I do it now I'll have to push the updated image to dockerhub everytime too so not going into that

Reply:
ah.